core/blockchain.go

```diff
@@ -162,6 +163,10 @@ func NewBlockChain(chainDb ethdb.Database, pow pow.PoW, mux *event.TypeMux) (*Bl
 	return bc, nil
 }
 
+func (self *BlockChain) SetConfig(vmConfig *vm.Config) {
```

Comment:
Also please mention that this method is not thread safe. I.e. after you start using the blockchain, you cannot call this method any more.